homeassistant/components/somfy/manifest.json

```diff
@@ -5,5 +5,5 @@
   "documentation": "https://www.home-assistant.io/integrations/somfy",
   "dependencies": ["http"],
   "codeowners": ["@tetienne"],
-  "requirements": ["pymfy==0.9.1"]
-}
+  "requirements": ["pymfy==0.9.3"]
```

Comment:
Optional: For what it's worth, i noticed this release has not yet been tagged on github.  I was trying to dig into other changes between version 0.9.1 and 0.9.3 since i've seen other folks do this when reviewing version bumps.

Reply:
You are right, I forgot to push the tag on the origin. 
See https://github.com/tetienne/somfy-open-api/compare/0.9.1...0.9.3